frontend/src/app/testResults/TestResultsList.test.tsx

```diff
@@ -1488,7 +1488,7 @@ describe("TestResultsList", () => {
     expect(await row.findByText("Colleer, Barde X")).toBeInTheDocument();
     expect(await row.findByText("DOB: 11/07/1960")).toBeInTheDocument();
     expect(await row.findByText("Negative")).toBeInTheDocument();
-    expect(await row.findByText("Facility 1")).toBeInTheDocument();
+    expect(row.queryByText("Facility 1")).not.toBeInTheDocument();
```

Comment:
I would prefer we add a test here for the happy path case on multiple facilities (ie asserting against the facility column/respective names showing up when we're passing 2+ facilities to the front end.) Don't want an inadvertent code change in the future to cause the facility column to not show up or the various facility names to appear incorrectly. Happy to explain more if needed!